src/components/20-molecules/policy-features/ui.test.js

```diff
@@ -0,0 +1,69 @@
+import { Selector } from 'testcafe';
+
+const host = process.env.TEST_HOST_STORYBOOK_URL || 'http://localhost:9999';
+const TAG = 'axa-policy-features';
+const CLASS = '.m-policy-features';
+const defaultWindowHeight = 1000;
+const huge2WindowWidth = 992;
+const hugeWindowWidth = 768;
+const large1WindowWidth = 576;
```

Comment:
With those sizes, you never test the mobile mode, because this goes from 0-575.

Moreover I would use the devices as terminology, but that's a personal preference. Like `mobileViewPortWidth`, `tabledViewPortWidth`, etc.

Reply:
I do not test xs explicitly, because large1 is the smallest width defined in the scss. Every screen size smaller than large1 has the same behavior.